package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "uk.co.compucorp.civicase",
+  "version": "2.0.0",
+  "description": "Civicase Extension based on Shoreditch theme",
```

Comment:
Lets take this from the `readme` file.
`The CiviCase extension is an overhaul of the CiviCase UI.  It provides a richer experience with more thoughtful layouts and interactions.`

Reply:
Updated.